packages/themes/__tests__/mixins-test.js

```diff
@@ -54,6 +57,10 @@ describe('_mixins.scss', () => {
     const { calls } = await render(`
       @import '../scss/themes';
 
+      $feature-flags: (
+        enable-css-custom-properties: false,
```

Comment:
(same as above) it seems setting the feature flag does not affect the test result, need to look more into this

the commented lines will cause the test to pass, but I think the issue is that the variables are becoming custom properties regardless of feature flag setting...